lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+  attr_reader :dishes
+
+
+  def initialize(dishes = { "pepperoni" => 7.50, "hawaiian" => 6.50, "vegetarian" => 5.50 })
+    @dishes = dishes
+  end
+
+
+
```

Comment:
This is a very minor gripe, but you probably should't be leaving this much white space between methods - a single line will do 🙂